src/utils/send.js

```diff
@@ -1,35 +1,61 @@
 "use strict";
 
-const https = require('https');
-
-// `data` is a JSON-stringified object.
-function httpsSend(data, requestConfigs) {
-  const req = https.request(requestConfigs, (response) => {
-    let responseData = '';
-    response.on('data', (chunk) => responseData += chunk);
-    response.on('end', () => console.log('Response body: ', responseData));
-  });
-
-  req.on('error', (err) => console.error('Error: ', err));
-  req.write(data);
-  req.end();
+import axios from "axios";
+import processEvent from "./process-event.js";
+import processUser from "./process-user.js";
+import getTimeStamp from "./get-timestamp.js";
+import stringifyObject from "./process-attributes.js";
+
+async function put(category, data, requestConfigs) {
```

Comment:
Suggestion: rename this to `post`, since that's the method that's being used instead of PUT.